definitions/npm/@react-navigation/core_v3.x.x/flow_v0.60.x-/core_v3.x.x.js

```diff
@@ -0,0 +1,745 @@
+// @flow
+
+declare module '@react-navigation/core' {
+  /**
+   * First, a bunch of things we would love to import but instead must
+   * reconstruct (mostly copy-pasted).
+   */
+
+  // This is a bastardization of the true StyleObj type located in
+  // react-native/Libraries/StyleSheet/StyleSheetTypes. We unfortunately can't
+  // import that here, and it's too lengthy (and consequently too brittle) to
+  // copy-paste here either.
+  declare type StyleObj =
+    | null
+    | void
+    | number
+    | false
+    | ''
+    | $ReadOnlyArray<StyleObj>
+    | { [name: string]: any };
+  declare type ViewStyleProp = StyleObj;
+  declare type TextStyleProp = StyleObj;
+  declare type AnimatedTextStyleProp = StyleObj;
+
+  // This one is too large to copy. Actual definition is in
+  // react-native/Libraries/Animated/src/nodes/AnimatedValue.js
+  declare type AnimatedValue = Object;
```

Comment:
You know that from Flow 0.75 this is an alias for `any`, right?

Reply:
Didn't know that! Thanks for the pointer.

What would you suggest doing in this case? The source type is [here](https://github.com/facebook/react-native/blob/master/Libraries/Animated/src/nodes/AnimatedValue.js#L74). My instinct is to type just the methods [listed in the documentation](http://facebook.github.io/react-native/docs/animatedvalue.html), though this will be a fairly brittle solution, and may require updating this libdef whenever React Native's Animated API changes.

Reply:
I'm afraid I don't have the right answer here, and that might even change when exact objects will become the default. I tend to use `{}` instead, but seems fragile as well.  
Maybe @jbrown215 could gives use some hints ;)